env0/data_sshkey.go

```diff
@@ -18,12 +18,14 @@ func dataSshKey() *schema.Resource {
 				Type:         schema.TypeString,
 				Description:  "the name of the ssh key",
 				Optional:     true,
+				Computed:     true,
```

Comment:
Adding computed fixes some plan issues.

With computed:

```
      + path                  = "misc/null-resource"
      + repository            = "https://github.com/tomer-landesman/templates"
      + ssh_keys              = [
          + (known after apply),
        ]
      + terraform_version     = "0.15.1"
      + type                  = "terraform"
    }

Without computed it's missing the id in the first apply
```
   + repository            = "https://github.com/tomer-landesman/templates"
      + ssh_keys              = [
          + {
              + "name" = "sshkey"
            },
        ]
      + terraform_version     = "0.15.1"
```

Reply:
For context data is something like this (with name):

```
data "env0_ssh_key" "data_ssh_key" {
  count = 1
  name  = "sshkey"

  depends_on = [env0_ssh_key.ssh_key]
}
```